src/bitriseAPI/capacityCheckStart.js

```diff
@@ -124,40 +117,39 @@ const capacityCheckStart = async () => {
   });
 
   const sessionsYouWillNeed =
-    WORKFLOW === "Workflow_Android"
-      ? Number(RESERVED_SESSIONS["Workflow_Android"]) +
-        Number(RESERVED_SESSIONS["Workflow_IOS"])
-      : Number(workflowData[WORKFLOW].RESERVED_SESSIONS);
+    workflow === "workflow_Android"
+      ? Number(RESERVED_SESSIONS["workflow_Android"]) +
+        Number(RESERVED_SESSIONS["workflow_IOS"])
+      : Number(workflowData[workflow].RESERVED_SESSIONS);
 
   if (occupiedSessions + sessionsYouWillNeed > SESSIONS_LIMIT) {
     console.log(
       chalk.red(
-        `Workflow ${WORKFLOW} will need ${sessionsYouWillNeed} sessions, so it should not be triggered at the moment :( Estimated sessions (with your build) ${
+        `workflow ${workflow} will need ${sessionsYouWillNeed} sessions, so it should not be triggered at the moment :( Estimated sessions (with your build) ${
           occupiedSessions + sessionsYouWillNeed
         }`
       )
     );
   } else {
     console.log(
       chalk.green(
-        `${WORKFLOW} will need ${sessionsYouWillNeed} sessions, so it can be triggered!!!`
+        `${workflow} will need ${sessionsYouWillNeed} sessions, so it can be triggered!!!`
       )
     );
-    let shouldTriggerBuild;
-    while (!shouldTriggerBuild) {
-      shouldTriggerBuild = readline.question(
-        `Would you like to trigger the ${WORKFLOW}? (y/n):\n`,
-        {
-          limit: YES_NO_OPTIONS,
-          limitMessage: `Type y or n!`,
-        }
-      );
-    }
+    const triggerQuestion = await inquirer.prompt([
+      {
+        name: "trigger",
+        message: `Would you like to trigger the ${workflow}?`,
+        type: "list",
+        choices: ["Yes", "No"],
+      },
+    ]);
```

Comment:
Since this kind of prompts are possibly quite common and the most used, could you please create a custom util that will accept the name and the message, and trigger the prompt?

Reply:
You are absolutely right!